service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/controller/utils/ReportItkHeaderParserUtil.java

```diff
@@ -46,6 +53,8 @@ private void setAddressList(ItkReportHeader header, Element headerElement) {
             .map(it -> it.attributeValue(URI_ATTRIBUTE))
             .findFirst();
 
+        itkValidator.checkItkOdsAndDosId(odsCode.orElse(null), dosServiceId.orElse(null));
```

Comment:
can we call itkValidator where other validators are called? setAddressList method is not a place where one would expect such validation to happen